tensorflow/core/kernels/conv_ops_test.cc

```diff
@@ -173,6 +173,61 @@ class FusedResizePadConvOpTest : public OpsTestBase {
 
     test::ExpectTensorNear<float>(unfused_tensors[0], fused_tensors[0], 1e-5);
   }
+
+  void CompareFusedPadOnlyAndSeparate(int input_width, int input_height,
+                                      int input_depth, int y_padding,
+                                      int x_padding, int filter_size,
+                                      int filter_count, string pad_mode,
+                                      int stride, string padding) {
+    auto root = tensorflow::Scope::NewRootScope();
+    using namespace ::tensorflow::ops;  // NOLINT(build/namespaces)
+
+    const size_t input_data_size = input_height * input_width * input_depth;
+    Tensor input_data(DT_FLOAT,
+                      TensorShape({1, input_height, input_width, input_depth}));
```

Comment:
you could use FillIota from tensor_testutil.h instead (if filling 0, ...  is okay).